src/sessions/SessionManager.ts

```diff
@@ -104,8 +107,8 @@ export class SessionManager {
     /**
      * Returns the session ID. If no session ID exists, one will be created.
      */
-    public getSession(): Session {
-        if (this.session.sessionId === NIL_UUID) {
+    public getSession(createIfNull = true): Session {
```

Comment:
issue: This flag variable is confusing. One alternative is to create a new method like `isSampled`.

Reply:
Refactored to add public method.